spec/CU-86ewrbrb0/ticket.md

```
@@ -0,0 +1,352 @@
+
+Design
+
+Key Concept: Unique Resolvers
+
+1 resolver = 1 unique resolver instance
+
+Even if two resolvers have the same username/name:version, if they have different configs, they are considered 2 different resolvers.
+
+Example:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)